src/hotspot/cpu/s390/c1_Runtime1_s390.cpp

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (c) 2016, 2021, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2016 SAP SE. All rights reserved.
+ * Copyright (c) 2016, 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2023, SAP SE. All rights reserved.
```

Comment:
As I understand it, only SAPers should touch the SAP copyright header. I recognize that this is a bit confusing because everyone updates the Oracle header.

If @RealLucy confirms this to be true, please keep the Oracle line update, and revert the SAP line.

Reply:
would you suggest that we add IBM copyright header here as well ?

Reply:
Here is the "common practice" for updating copyright headers:

- Everybody updates the Oracle copyright line if there was a change, however small, in the file.
- Everybody can (but doesn't have to) update the SAP copyright line if there was a change.
- Please note: the SAP copyright does not have a comma after the last year.
- Adding the own company's copyright line is regarded acceptable only in case of major contributions/changes.